ghost/i18n/locales/es/portal.json

```diff
@@ -27,10 +27,10 @@
     "Cancel subscription": "Cancelar suscripción",
     "Cancellation reason": "Razón de cancelación",
     "Change": "Cambiar",
-    "Change plan": "",
+    "Change plan": "Cambiar plan",
     "Check spam & promotions folders": "Comprueba la carpeta de spam y promociones",
     "Check with your mail provider": "Comprueba con tu proveedor de correo",
-    "Check your inbox to verify email update": "",
+    "Check your inbox to verify email update": "Revisa tu correeo electrónico para actualizar el email",
```

Comment:
another spelling error

Reply:
In spanish it is accepted the same as correo electrónico same as email. De acuerdo con la Real Academia Española (RAE), la palabra "e-mail" es un extranjerismo crudo y se recomienda usar la forma española "correo electrónico". Si se usa en español, debe escribirse en cursiva. I wrote correo electrónico then email to not sound redundant.

Reply:
Oh god. Got that, yes, it is "Revisa tú bandeja de entrada para actualizar el correo electrónico"